api/TextTrack.json

```diff
@@ -657,7 +657,7 @@
               "version_added": "8"
             },
             "safari_ios": {
-              "version_added": "13"
+              "version_added": "14"
```

Comment:
This is the `sourceBuffer` attribute. Is `partial_implementation` and a note about iPadOS as in https://github.com/mdn/browser-compat-data/blob/master/api/SourceBuffer.json needed here too, or is it actually exposed on iOS? If it is, I would assume it doesn't really do anything.

Reply:
This is weird, it's exposed on both iOS and iPadOS 13...  What is going on with the versions?

Reply:
Was this a bug with the update-bcd script? If so can you file a bug and investigate so it doesn't happen again?